go.sum

```diff
@@ -16,13 +16,16 @@ github.com/AlecAivazis/survey/v2 v2.0.7/go.mod h1:mlizQTaPjnR4jcpwRSaSlkbsRfYFEy
 github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
+github.com/Djarvur/go-err113 v0.0.0-20200511133814-5174e21577d5/go.mod h1:4UJr5HIiMZrwgkSPdsjy2uOQExX/WEILpIrO9UPGuXs=
```

Comment:
this files seems weird, is adding a bunch of stuff that we don't bundle with, can you please do `go mod tidy` to clean unused dependencies?

Reply:
This is caused by [digest](https://github.com/mongodb-forks/digest) passing down `golang-ci` as a required mod.
I opened a PR to clean it up: https://github.com/mongodb-forks/digest/pull/5